app/frontend/src/features/profile/view/leaveReference/formSteps/Text.tsx

```diff
@@ -0,0 +1,105 @@
+import { Typography, useMediaQuery, useTheme } from "@material-ui/core";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import TextBody from "components/TextBody";
+import TextField from "components/TextField";
+import {
+  NEXT,
+  PUBLIC_ANSWER,
+  REFERENCE_FORM_HEADING_FRIEND,
+  REFERENCE_FORM_HEADING_HOSTED,
+  REFERENCE_FORM_HEADING_SURFED,
+  TEXT_EXPLANATION,
+} from "features/profile/constants";
+import {
+  ReferenceContextFormData,
+  useReferenceData,
+} from "features/profile/view/leaveReference/ReferenceDataContext";
+import {
+  ReferenceFormProps,
+  useReferenceStyles,
+} from "features/profile/view/leaveReference/ReferenceForm";
+import React from "react";
+import { Controller, useForm } from "react-hook-form";
+import { useHistory, useParams } from "react-router-dom";
+import { leaveReferenceBaseRoute } from "routes";
+
+export default function Text({ user }: ReferenceFormProps) {
+  const history = useHistory();
+  const classes = useReferenceStyles();
+  const theme = useTheme();
+  const isMdOrWider = useMediaQuery(theme.breakpoints.up("md"));
+  const { referenceType, hostRequest } = useParams<{
+    referenceType: string;
+    hostRequest?: string;
+  }>();
+  const { data, setValues } = useReferenceData()!;
+  const { control, handleSubmit, errors } = useForm<ReferenceContextFormData>({
+    defaultValues: {
+      text: data.text,
+    },
+  });
+
+  const onSubmit = (values: ReferenceContextFormData) => {
+    setValues(values);
+    hostRequest
+      ? history.push(
+          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/${hostRequest}/submit`
+        )
+      : history.push(
+          `${leaveReferenceBaseRoute}/${referenceType}/${user.userId}/submit`
+        );
+  };
+
+  return (
+    <form className={classes.form} onSubmit={handleSubmit(onSubmit)}>
+      {referenceType === "friend" && (
+        <Typography variant="h2">
+          {REFERENCE_FORM_HEADING_FRIEND}
+          {user.name}
+        </Typography>
+      )}
+      {referenceType === "hosted" && (
+        <Typography variant="h2">
+          {REFERENCE_FORM_HEADING_HOSTED}
+          {user.name}
+        </Typography>
+      )}
+      {referenceType === "surfed" && (
+        <Typography variant="h2">
+          {REFERENCE_FORM_HEADING_SURFED}
+          {user.name}
+        </Typography>
+      )}
+      <TextBody className={classes.text}>{TEXT_EXPLANATION}</TextBody>
+      <TextBody className={classes.text}>{PUBLIC_ANSWER}</TextBody>
+      {errors && errors.text?.message && (
+        <Alert className={classes.alert} severity="error">
+          {errors.text.message}
+        </Alert>
+      )}
+      <div className={classes.card}>
+        <Controller
+          render={({ onChange }) => (
+            <TextField
```

Comment:
This doesn't need a controller, just set `name` on the `TextField` directly and do `inputRef={register}`

Reply:
That should also fix the default value which isn't currently working (because controller doesn't have a way to set it on the rendered component)